extractor/src/main/java/org/schabi/newpipe/extractor/services/media_ccc/linkHandler/MediaCCCStreamLinkHandlerFactory.java

```diff
@@ -9,10 +8,12 @@
 import java.net.URL;
 
 public class MediaCCCStreamLinkHandlerFactory extends LinkHandlerFactory {
+    public static final String VIDEO_API_ENDPOINT = "https://api.media.ccc.de/public/events/";
+    private static final String VIDEO_PATH = "https://media.ccc.de/v/";
 
     @Override
     public String getId(String urlString) throws ParsingException {
-        if (urlString.startsWith("https://api.media.ccc.de/public/events/") &&
+        if (urlString.startsWith(VIDEO_API_ENDPOINT) &&
                 !urlString.contains("?q=")) {
             return urlString.substring(39); //remove api…/public/events part
         }
```

Comment:
Why are you using a regex for conferences while using `substring()` for API URLs here and using `URL#getPath()` + `substring()` for non-API URLs here?

Reply:
I left everything as it was before, I only replaced the hardcoded url. Will fix this